cloud/pkg/devicecontroller/controller/downstream.go

```diff
@@ -647,6 +632,12 @@ func (dc *DownstreamController) deviceUpdated(device *v1alpha2.Device) {
 						klog.Errorf("Failed to send deviceTwin message %v due to error %v", msg, err)
 					}
 				}
+				// distribute device model
+				if isDeviceStatusUpdated(&cachedDevice.Status, &device.Status) ||
+					isDeviceDataUpdated(&cachedDevice.Spec.Data, &device.Spec.Data) {
+					dc.sendDeviceModelMsg(device, model.UpdateOperation)
```

Comment:
when the device is updated, why do we need send the device model msg?

Reply:
the device model is not binded to the node and cloudcore does not know which node the device model info should be sent to. the device is binded to the node.